.github/ISSUE_TEMPLATE/feature.yaml

```diff
@@ -0,0 +1,41 @@
+name: "🚀 Feature"
+description: "Submit a proposal for a new feature"
+title: "🚀 Feature: "
+labels: [feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
```

Comment:
Please update this part as showed in our template. We cannot thank for bug report in feature request.